### PR TITLE
feat(website): add link to open Github issue

### PR DESCRIPTION
**Pull request by massao**
Add link on PageContentHeader to give about the Component.
* The link will use the `component-feedback.md` issue template.
* Add the component name to the title

### CODE REVIEW

File: packages/website/components/PageContent/PageContentHeader.tsx [hunk 1]
```diff
@@ -57,15 +69,21 @@ export function PageContentHeader({
       </DisplayText>
 
       {github && (
```
Comment (gui-santos) [inline packages/website/components/PageContent/PageContentHeader.tsx:71]: Shouldn't we remove this condition now?

This prop is not used by the things that are rendered if the condition is true

**Review by gui-santos — APPROVED** [review]
Looks great! I just have one question

**Review by bgutsol — APPROVED** [review]
👍

File: packages/website/components/PageContent/PageContentHeader.tsx (hunk 1 above)
Comment (massao) [reply packages/website/components/PageContent/PageContentHeader.tsx:71]: Nice catch, forgot to remove it

**Pull request merged**